### DISCUSSION TITLE
How to add BetaPERT distribution?

### DISCUSSION THREAD

**Discussion by luxspes** [oracle/sandwood#5] [Ideas]
How to add BetaPERT distribution? (something like https://github.com/oxinabox/ProjectManagement.jl )

**Comment by Daniel-Goodman** [comment]
We do not currently have tooling to do this automatically. Clearly writing this out verbatim by hand is not practical so that leaves you with, 2 options:
1. Create a script that will automatically generate the require Sandwood code for the model and then compile that model.
2. Write a piece of Java code to generate the required API calls for the model and then compile the resulting DAG. I suspect this is the direction you are thinking about going given your [other question](https://github.com/oracle/sandwood/discussions/6). 

If you want to know more about the API calls, api.java files in the expected data resources for the compiler tests contain examples of the api calls generate for a range of different models found in the test inputs. If you want extra help with this please reach out.
